fakelibusb_test.go

```diff
@@ -87,6 +87,8 @@ type fakeLibusb struct {
 	mu sync.Mutex
 	// fakeDevices has a map of devices and their descriptors.
 	fakeDevices map[*libusbDevice]*fakeDevice
+	// fakeSysDevices keeps the order of devices to be accessd by wrapSysDevice
+	fakeSysDevices map[uintptr]*libusbDevice
```

Comment:
actually, perhaps just "sysDevices" - this is already inside the fakeLibusb, so "fake" stutters a bit. "fakeDevices" is ok because the type is called "fakeDevice", although perhaps it should also be renamed "devices"...

Reply:
You're the boss, but I'd argue that either both should be changed or neither - it's much more readable to see these two named in the same way. You immediately go "ah, right, these are just some mappings of the fakes"

Reply:
I'd go with "both" then :)

Reply:
I understand this as "change both" :) 
Will update